### PR TITLE
DragHandle icon; compl palette derived color

### PR DESCRIPTION
**Pull request by strouhaljan**
JIRA: BB-2824

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3073/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4723/ : FAILURE in 2m 41s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3074/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4724/ : SUCCESS in 12m 32s

File: libs/sdk-ui-kit/src/Icon/icons/DragHandle.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import { IIconProps } from "../typings";
+
+/**
+ * @internal
+ */
+export const DragHandle: React.FC<IIconProps> = ({ className, color }) => {
+    return (
+        <svg
+            className={classNames(className, "sdk-ui-kit-icon")}
```
Comment (scavnickyj) [inline libs/sdk-ui-kit/src/Icon/icons/DragHandle.tsx:13]: What is this class for?

File: libs/sdk-ui-kit/src/Icon/Icon.tsx [hunk 2]
```diff
@@ -28,9 +28,12 @@ export const Icon: React.FC<IIconProps> = (props) => {
         case "Undo":
         case "Home":
         case "BurgerMenu":
-        case "Rows": {
+        case "Rows":
+        case "DragHandle": {
             const IconComponent = Icons[name];
-            return <IconComponent name={name} color={color} colorPalette={colorPalette} />;
+            return (
+                <IconComponent className={className} name={name} color={color} colorPalette={colorPalette} />
```
Comment (scavnickyj) [inline libs/sdk-ui-kit/src/Icon/Icon.tsx:35]: Maybe we could also introduce something like `classNames(className, "s-{name}.toLowerCase()")`. What do you think?

File: libs/sdk-ui-kit/src/Icon/icons/DragHandle.tsx (hunk 1 above)
Comment (strouhaljan) [reply libs/sdk-ui-kit/src/Icon/icons/DragHandle.tsx:13]: It is for further CSS positioning etc. I didn't want to wrap it into the `<div>` just for that purpose and I suppose we might want to be able to select the icon with the CSS selector in the future anyway.

Comment (strouhaljan) [reply libs/sdk-ui-kit/src/Icon/icons/DragHandle.tsx:13]: (perhaps, we should alter all the existing components and add this property as well)

File: libs/sdk-ui-kit/src/Icon/Icon.tsx (hunk 2 above)
Comment (strouhaljan) [reply libs/sdk-ui-kit/src/Icon/Icon.tsx:35]: Do you think we might need to test the icons? I am thinking about existence of an icon, but still not convinced we should add the `s-` class just for this very occasional use case.

Comment (strouhaljan) [reply libs/sdk-ui-kit/src/Icon/Icon.tsx:35]: We decided to omit `s-` class name for now.

File: libs/sdk-ui-kit/src/Icon/icons/DragHandle.tsx (hunk 1 above)
Comment (strouhaljan) [reply libs/sdk-ui-kit/src/Icon/icons/DragHandle.tsx:13]: We decided to omit `sdk-ui-kit-icon` class name for now.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3077/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4728/ : SUCCESS in 12m 38s

**Comment by scavnickyj** [comment]
extended test

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/781/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3080/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/781/ : SUCCESS in 27m 55s

**Review by scavnickyj — APPROVED** [review]
LGTM

**Comment by strouhaljan** [comment]
extended test

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/782/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3081/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/782/ : SUCCESS in 16m 21s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- gooddata-ui-sdk-fossa-scanning-zuul-el7 : SKIPPED (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1187/ : FAILURE in 18s
- gooddata-ui-sdk-unit-tests : SKIPPED
- gooddata-ui-sdk-sonarqube-scan-zuul-el7 : SKIPPED (non-voting)

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1209

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/920/ : SUCCESS in 43s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1188/ : SUCCESS in 22s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4731/ : SUCCESS in 15m 25s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/302/ : SUCCESS in 8m 47s (non-voting)

**Pull request merged**